bin/run-command.sh

```diff
@@ -31,28 +31,28 @@ echo "  ENVIRONMENT_VARIABLES=$ENVIRONMENT_VARIABLES"
 echo
 
 # Use the same cluster, task definition, and network configuration that the application service uses
-CLUSTER_NAME=$(terraform -chdir=infra/$APP_NAME/service output -raw service_cluster_name)
-SERVICE_NAME=$(terraform -chdir=infra/$APP_NAME/service output -raw service_name)
+CLUSTER_NAME="$(terraform -chdir="infra/$APP_NAME/service" output -raw service_cluster_name)"
+SERVICE_NAME="$(terraform -chdir="infra/$APP_NAME/service" output -raw service_name)"
```

Comment:
See if shellcheck complains about this:
```suggestion
CLUSTER_NAME=$(terraform -chdir="infra/$APP_NAME/service" output -raw service_cluster_name)
SERVICE_NAME=$(terraform -chdir="infra/$APP_NAME/service" output -raw service_name)
```

Reply:
yeah doesn't complain about this, but seems like i need to keep the ones in current-account-id.sh, current-account-alias.sh